persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java

```diff
@@ -506,7 +506,7 @@ public void bundleChanged(BundleEvent event) {
     }
 
     private String getMonthlyIndexName(Date date) {
-        String d = new SimpleDateFormat("-YYYY-MM").format(date);
+        String d = new SimpleDateFormat("-yyyy-MM").format(date);
```

Comment:
We execute this code quite ofter, and creating SimpleDateFormat could be expensive. However, we can't re-use it here I think, since it's not thread safe. 

I can optimize this code by changing it to this if you prefer:
`String d = "-" + (d.getYear() + 1900) + "-" + (d.getMonth() + 1)`